ios-swift/HelloWorldSwift/AppDelegate.swift

```diff
@@ -29,13 +30,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let settings = UIUserNotificationSettings(forTypes: .Alert | .Badge | .Sound, categories: nil)
         UIApplication.sharedApplication().registerUserNotificationSettings(settings)
         UIApplication.sharedApplication().registerForRemoteNotifications()
+        
+        // Send metrics when app is launched due to push notification
+        AGPushAnalytics.sendMetricsWhenAppLaunched(launchOptions)
+        
+        // When the app is launched due to a push notification
```

Comment:
should we be explict here, what's being done below, when app is opened due to a push ? 

The comment above the `AGPushAnalytics.sendMetricsWhenAppLaunched` says also app opened due to push